crates/bevy_app/src/app.rs

```diff
@@ -43,6 +43,7 @@ bevy_utils::define_label!(AppLabel);
 ///    println!("hello world");
 /// }
 /// ```
+#[must_use = "`App::run` is not being called. Any systems and/or plugins added to this `App` will not run."]
```

Comment:
```suggestion
#[must_use = "You have not called `App::run` or `App::update`. Any systems and/or plugins added to this `App` will not run."]
```

I often use `App::update` by itself in tests.